modules/heat_transfer/test/tests/heat_conduction/2d_quadrature_gap_heat_transfer/tests

```diff
@@ -76,4 +76,31 @@
     design = "/actions/ThermalContactAction.md"
     issues = "#13043"
   [../]
+
+  [perfect_prereq]
+    type = 'CheckFiles'
+    prereq = perfect
+    input = 'perfect.i'
+    delete_output_before_running = false
```

Comment:
```suggestion
```
I dont see why?  If you dont delete output, you could be using the output from another test

Reply:
Hmm, This is a two part test, but usually you need that line in the second test, not the first and only if there is an explicit overlap on which files are checked. I agree this line shouldn't need to be here.

Reply:
@permcody  I don't get this. I had before uploaded the specific files and you recommended to use this option instead. The test suite fails without assuring the files are there. There is the option of setting a test as prerequisite, but not sure it was sufficient.

Reply:
prereq is needed imo. I have not played with the test more than that